nebula-logger/core/main/log-management/classes/LogEntryHandler.cls

```diff
@@ -44,16 +45,22 @@ public without sharing class LogEntryHandler extends LoggerSObjectHandler {
     }
 
     private void setApexClassFields() {
-        List<String> apexClassNames = new List<String>();
-        List<LogEntry__c> apexLogEntries = new List<LogEntry__c>();
+        Set<String> apexClassNames = new Set<String>();
+        List<LogEntry__c> apexClassLogEntries = new List<LogEntry__c>();
         for (LogEntry__c logEntry : this.logEntries) {
-            if (logEntry.OriginType__c == 'Apex' && String.isNotBlank(logEntry.OriginLocation__c)) {
-                // OriginLocation__c contains the class name + method name. Examples:
-                // MyClass.someMethod
-                // MyClass.MyInnerClass.someMethod
-                String apexClassName = logEntry.OriginLocation__c.substringBefore('.');
-                apexClassNames.add(apexClassName);
-                apexLogEntries.add(logEntry);
+            Boolean isApexClassLogEntry = false;
+            if (logEntry.OriginSourceMetadataType__c == 'ApexClass' && String.isNotBlank(logEntry.OriginSourceApiName__c)) {
```

Comment:
any point in caching the result of ApexClass.SObjectType.getDescribe().getName() for this and the below check?

Reply:
One amazing point - no more hardcoded strings!!!! 🥳 🎉 🤩  🥇 🏆 👏 

I'll switch to using `getName()`!